pages/08-freunde.md

```diff
@@ -4,18 +4,9 @@ description: Verwandte Projekte der Genossenschaft
 permalink: /freunde/
 ---
 
-# Hofkäserei Im Basi
-
-Anita und Fabian und ihre lieben Kühe sind unsere Milchproduzenten.
-Sie bewirtschaften ihren Milch- und Ackerbaubetrieb nach Bio-Suisse
-Richtlinien. Die Milchprodukte sind in ihrem Hofladen zu finden.
-
-<http://www.hofkaeserei-imbasi.ch>
-
-
 # ortoloco
 
-Die Grundidee für {{ site.info.shortname }} stammt von ortoloco, der selbstverwalteten Gemüsekooperative auch aus Dietikon.
+Die Grundidee für basimilch stammt von ortoloco, der selbstverwalteten Gemüsekooperative auch aus Dietikon.
```

Comment:
Zur Info: `{{ site.info.shortname }}` wird in der Webseite mit `basimilch` ersetzt. Es so zu machen, hat den Vorteil, dass wir einheitlich steuern können, wie der Name `basimilch` dargestellt wird (z.B. fett, oder nicht fett...).
Es gibt ein paar solche Variabeln, zB:
- `{{ site.info.subtitle }}` wird zu `Die kooperative Käserei in Dietikon`, oder
- `{% include info_email_link.html %}` wird immer zu einem Info-Email-Link.

:blush: :+1: